### ISSUE TITLE
Documentation in Dash

### ISSUE THREAD

**Issue by roennow** [tailwindcss/discuss#66]
I can't be the only one who loves Dash for documentation. Is it possible/easy to compile the documentation and add it do dash?

**Comment by aparajita** [comment]
I'd love Dash docs too. I think you'd have to scrape the docs to html (best to use something like [SiteSucker](https://itunes.apple.com/us/app/sitesucker/id442168834?mt=12)) and then [convert to Dash format](https://kapeli.com/docsets#dashDocset).

**Comment by m1guelpf** [comment]
While we are at this, it would also be awesome to PR Tailwind to devdocs.io

**Comment by brentdodell** [comment]
Any progress on this? New to Tailwind, but would love to see the docs in Dash.

**Comment by tnorthcutt** [comment]
Corresponding issue in the Dash user contributions repo: https://github.com/Kapeli/Dash-User-Contributions/issues/1870

**Comment by bkbooth** [comment]
Is anyone working on this? I might take a look at it

**Comment by aparajita** [comment]
Apparently not. I gave it a shot and it wasn't straightforward. You'll probably have to ask Adam to modify the doc generation to get better cross linking.

**Comment by bkbooth** [comment]
I finally found some time to work on this, it's just waiting to be approved and merged into the Kapeli/Dash-User-Contributions repo.

My repo for the scraper + docs builder is here: https://github.com/bkbooth/dash-tailwindcss-docs (I'll update the README with a little more details soon)

I'm open to suggestions and/or PR's. It was a little difficult to come up with types (https://kapeli.com/docsets#supportedentrytypes) for the pages, for now I've used "Guide" for the pages under "Getting Started" in the Tailwind docs, "Sample" for the "Component Examples" pages and "Section" for everything else.

Also, there are two pages called "Display", one under "Layout" and another under "Flexbox", let me know if you have ideas for better ways to name the pages in Dash. I thought of prefixing the section title before the page title, but I don't like that as a hard rule, maybe just the pages under "Flexbox"?

**Comment by aparajita** [comment]
@bkbooth Adam is planning to completely redo the docs in VuePress in the coming weeks. You might want to hold off on too much more work on this until he does that.

**Comment by bkbooth** [comment]
@aparajita thanks for heads up. Sounds like it'll probably take a little bit of work to update the scraper once the new docs site is finished.
